packages/design-system/src/components/css-value-list-item.stories.tsx

```diff
@@ -0,0 +1,165 @@
+import { css, styled, theme } from "../stitches.config";
+import { CssValueListItem } from "./css-value-list-item";
+import { Label } from "./label";
+import { SmallToggleButton } from "./small-toggle-button";
+import {
+  EyeconOpenIcon,
+  EyeconClosedIcon,
+  SubtractIcon,
+} from "@webstudio-is/icons";
+import * as React from "react";
+import { SmallIconButton } from "./small-icon-button";
+import {
+  FloatingPanelPopover,
+  FloatingPanelPopoverContent,
+  FloatingPanelPopoverTrigger,
+} from "./floating-panel-popover";
+import { StorySection, StoryGrid } from "./storybook";
+
+export default {
+  component: CssValueListItem,
+  args: { hidden: false, labelColor: "default", focused: false },
+  argTypes: {
+    hidden: { control: "boolean" },
+    focused: { control: "boolean" },
+    labelColor: {
+      options: ["default", "preset", "local", "remote"],
+      type: "inline-radio",
+    },
+  },
+};
+
+const Thumbnail = styled("div", {
+  width: theme.spacing[10],
+  height: theme.spacing[10],
+  backgroundImage: "linear-gradient(yellow, red)",
+});
+
+const ListItem = (props: {
+  hidden: boolean;
+  labelColor: "default" | "preset" | "local" | "remote";
+  state: undefined | "open";
```

Comment:
just optional works too
```suggestion
  state?: "open";
```

Reply:
It's a storybook and I want to explicitly pass undefined to be sure I checked most variants.
So its intented
<img width="263" alt="image" src="https://user-images.githubusercontent.com/5077042/221596296-2547018c-a37e-480c-b220-1044822b5aec.png">